src/sonic-yang-models/yang-models/sonic-interface.yang

```diff
@@ -45,9 +53,8 @@ module sonic-interface {
 				}
 
 				leaf vrf_name {
-					type string {
-						pattern "Vrf[a-zA-Z0-9_-]+";
-						length 3..255;
+					type leafref {
```

Comment:
We need similar changes in sonic-loopback-interface.yang

Reply:
Addressed